src/test/packages/podinfo/refs/zarf.yaml

```diff
@@ -0,0 +1,25 @@
+kind: ZarfPackageConfig
+metadata:
+  name: podinfo
+  version: 0.0.2
+
+components:
+  - name: podinfo
+    required: true
+    charts:
+      - name: podinfo
+        version: 6.6.2
+        namespace: podinfo
+        url: https://github.com/stefanprodan/podinfo.git
+        gitPath: charts/podinfo
+    images:
+      - ghcr.io/stefanprodan/podinfo:6.6.2
```

Comment:
Same comment as above, make a note that we are using a different version of podinfo and why